src/plugins/gui/providers/mtpelerinProvider.ts

```diff
@@ -264,41 +264,39 @@ type GetQuoteParams = (GetSourceQuoteParams | GetDestQuoteParams) & {
 
 type WidgetParams = (WidgetBuyParams | WidgetSellParams) & {
   _ctkn: string
-  lang?: string // 2 characters language code ('fr'|'en')
-  primary?: string // Primary color (hexadecimal encoded)
-  addr: string // Wallet address
-  code: string // Random 4 digit code from 1000-9999
-  hash: string // Hash of signature
-  net: string // Default network
-  type: 'webview' // Integration type ('web'|'popup'|'webview')
+  lang?: string /** 2 characters language code ('fr'|'en') */
```

Comment:
Comments need to be above the keys to show in hovers

Reply:
ack. waste of space.